devices/Dac63004/Dac63004.cs

```diff
@@ -0,0 +1,270 @@
+        /// <param name="mode">The funcional mode to put the channel in.</param>
+        /// <exception cref="NotSupportedException">If the <paramref name="mode"/> is not supported.</exception>"
+        public void ConfigChannelMode(
+            Channel channel,
+            Mode mode)
+        {
+            var currentConfig = ReadFromRegister(Register.Reg1F_CommonConfig, 2);
+            ushort newConfig;
+
+            if (mode == Mode.CurrentOutput)
+            {
+                newConfig = ConfigPowerUpCurrentOutChannel0Mask | ConfigPowerDownVoltageOutChannel0Mask;
+            }
+            else if (mode == Mode.VoltageOutput)
+            {
+                newConfig = ConfigPowerDownCurrentOutChannel0Mask | ConfigPowerUpVoltageOutChannel0Mask;
+            }
+            else
+            {
+                throw new NotSupportedException();
+            }
+
+            newConfig <<= (byte)channel * ConfigChannelOffset;
+
+            currentConfig[0] &= (byte)~newConfig;
+            currentConfig[1] &= (byte)~(newConfig >> 8);
+
+            WriteToRegister(Register.Reg1F_CommonConfig, currentConfig);
+        }
+
+        /// <summary>
+        /// Configures the DAC channel funcional mode.
+        /// </summary>
+        /// <param name="channel">The <see cref="Channel"/> to configure.</param>
+        /// <param name="gain">The <see cref="VoutGain"/> to configure the <paramref name="channel"/> to.</param>
+        public void ConfigChannelVoutGain(
+            Channel channel,
+            VoutGain gain)
+        {
+            Register registerAddress = DacVoutConfigFromChannel(channel);
+            var currentConfig = ReadFromRegister(registerAddress, 2);
+
+            currentConfig[1] &= (byte)gain;
+
+            WriteToRegister(registerAddress, currentConfig);
+        }
+
+        /// <summary>
+        /// Data for DAC output.
+        /// </summary>
+        /// <param name="channel">The <see cref="Channel"/> to set data to.</param>
+        /// <param name="value">The value to configure the data register for the <paramref name="channel"/>.</param>
+        public void ConfigChannelVoutGain(
+            Channel channel,
+            int value)
+        {
+            byte[] buffer = new byte[2];
+
+            Register registerAddress = Register.Reg19_DAC0_Data + (byte)channel;
+
+            // Data are in straight-binary format. MSB left-aligned.
+            var data = value << 4;
+
+            // split and fill buffer
+            buffer[1] = (byte)(data >> 8);
+            buffer[0] = (byte)(data & 0xFF);
+
+            WriteToRegister(registerAddress, buffer);
+        }
+
+        /// <inheritdoc/>
+        public void Dispose()
+        {
+            _i2cDevice?.Dispose();
+            _i2cDevice = null;
+        }
+
+        /// <summary>
+        /// Read part information to perform a sanity check for the correct part.
+        /// </summary>
+        private void ReadPartInformation()
```

Comment:
Rather than `Read` better use `Check` as it's what it does. You check that the device is the proper one and throw if not.
Also use `Device` rather than `Part`, it's more explicit.